src/EndOfYearBanner/index.tsx

```diff
@@ -127,26 +120,21 @@ const TheGuardianIn2020Banner: React.FC<Props> = (props: Props) => {
                     <p css={styles.paragraph}>
                         {body}
                         <br />
```

Comment:
Should the `<br/>` be included conditionally only if the `boldText` prop is supplied?

Reply:
Yep this is done now.